channels/stable

```diff
@@ -58,7 +58,7 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
-  - range: ">=1.18.0-alpha.1"
+  - range: ">=1.18.0"
```

Comment:
Please do this change in the alpha channel also.

Reply:
sure thing, in the same PR or should I open a new PR for those?

Reply:
Same PR would be ok.

Reply:
done